File: limiteMEI-angular/src/app/enums/natureza.receita.enum.ts
export enum NaturezaReceitaEnum {
  COMERCIO = 'COMERCIO',
  INDUSTRIA = 'INDUSTRIA',
  SERVICOS = 'SERVICOS'
}

export const NATUREZA_RECEITA_LABELS: Record<NaturezaReceitaEnum, string> = {
  [NaturezaReceitaEnum.COMERCIO]: 'Comércio',
  [NaturezaReceitaEnum.INDUSTRIA]: 'Indústria',
  [NaturezaReceitaEnum.SERVICOS]: 'Serviços'
};

File: limiteMEI-api/src/main/java/com/limiteMEI/limiteMEI/enums/NaturezaReceitaEnum.java
package com.limiteMEI.limiteMEI.enums;

import lombok.Getter;

@Getter
public enum NaturezaReceitaEnum {
    COMERCIO("Comércio"),
    INDUSTRIA("Indústria"),
    SERVICOS("Serviços");

    private final String nome;

    NaturezaReceitaEnum(String nome) {
        this.nome = nome;
    }
}
